zuul.d/edpm_periodic.yaml

```diff
@@ -51,6 +51,9 @@
     vars:
       cifmw_repo_setup_branch: antelope
       cifmw_set_openstack_containers_namespace: podified-{{ cifmw_repo_setup_branch }}-centos9
+      cifmw_run_test_role: test_operator
+      cifmw_test_operator_tempest_include_list: |
+        tempest.scenario.test_server_basic_ops.TestServerBasicOps
```

Comment:
this is what we have in the current job runs that use the tempest role, e.g. there
https://logserver.rdoproject.org/openstack-periodic-container-antelope-centos9/review.rdoproject.org/rdo-jobs/master/periodic-podified-edpm-baremetal-antelope-ocp-crc/61fa45a/ci-framework-data/tests/tempest/include.txt 

```
tempest.scenario.test_server_basic_ops.TestServerBasicOps.test_server_basic_ops

```